serverless/sql-databases/reference-content/known-differences.mdx

```diff
@@ -14,9 +14,9 @@ categories:
   - sql-databases
 ---
 
-Serverless SQL Database is based on PostgreSQL engine and thus provides standard PostgreSQL compatibility for most PostreSQL features and commands. Due to the nature of a managed service, autoscaling support and connection pooling support, some features are not currently supported. This means either the corresponding SQL command will not be authorized, or will not behave similarly to a standard PostgreSQL instance.
+Serverless SQL Database is based on PostgreSQL engine and thus provides standard PostgreSQL compatibility for most PostgreSQL features and commands. Due to the nature of a managed service, autoscaling support and connection pooling support, some features are not currently supported. This means either the corresponding SQL command will not be authorized, or will not behave similarly to a standard PostgreSQL instance.
 
-If you require strict PostgreSQL features compatibility, you can use [Managed database for PostgreSQL](link to managed database).
+If you require strict compatibility with all PostgreSQL features, you can use [Managed Databases for PostgreSQL](/managed-databases/postgresql-and-mysql/quickstart/) instead.
 
 ## SQL Features
```

Comment:
Suggestion "Unsupported SQL Features" (easier to read / eye catching when going through the page quickly)